Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/cherry_pick.py

```diff
@@ -86,11 +87,12 @@ def main(cls, args, repository, **kwargs):
         else:
             cherry_pick_string = '{} ({})'.format(commit, commit.hash[:commit.HASH_LABEL_SIZE])
 
+        env = os.environ.copy()
```

Comment:
Can we use specific env. variables or do we need to copy the entire environment?
Not sure if there can be any unintended side-effects because of different users having different/unexpected env. variables.

Reply:
While I could certainly make this work for my specific bug by only forwarding certain environment variables, I believe we need to copy the whole environment. My rationale for this is that `git-webkit cherry-pick` is basically supposed to be `git cherry-pick`, but with a better commit message. For that to be true, `git-webkit cherry-pick` really needs to invoke `git cherry-pick` exactly as the user would (which would include the current environment)

My specific bug is about `PATH` not being forwarded, so we're missing something our commit message generator needs. Which shows that `git` itself is forwarding environment variables.

Reply:
ok. make sense.